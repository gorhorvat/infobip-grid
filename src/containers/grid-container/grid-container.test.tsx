import React from "react";
import { shallow } from 'enzyme';
import GridContainer from "./grid-container";
import toJson from 'enzyme-to-json';

describe('GridContainer', () => {
  function setup() {
    return shallow(<GridContainer />);
  }

  it('Renders without crashing', () => {
    // Arrange
    const wrapper = setup();

    // Assert
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});