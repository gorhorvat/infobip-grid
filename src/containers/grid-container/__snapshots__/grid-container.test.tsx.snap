// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridContainer Renders without crashing 1`] = `
<Grid
  columns={10}
  rows={9}
  size="large"
/>
`;
